packages/helpers/src/cms/getCmsBreadcrumbs.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Build the breadcrumbs for the CMS page
+ *
+ * @param page
+ * @returns
+ */
+export function getCmsBreadcrumbs<
```

Comment:
not sure about this helper - it's not creating breadcrumbs, just displaying entity translated property, when would that be useful for the users?